Weather Man/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.txt
+/weatherdata/
```

Comment:
pdf, cache files should also be local only, only code files should be on the github